airflow/models/taskinstance.py

```diff
@@ -1213,10 +1213,11 @@ def _run_finished_callback(
     if callbacks:
         callbacks = callbacks if isinstance(callbacks, list) else [callbacks]
         for callback in callbacks:
+            callback_name = qualname(callback).split(".")[-1]
             try:
+                log.info("Executing %s callback", callback_name)
```

Comment:
This should be put outside of the try block.

Reply:
Also since we only use the last component, we can just use `callback.__name__` for the same effect (or just `callback`). I know this is inherited from existing code.

Reply:
> Also since we only use the last component, we can just use `callback.__name__` for the same effect (or just `callback`). I know this is inherited from existing code.

when changing to `callback.__name__` works.

just to let you know, changing to `callback` didn't work:
<img width="689" alt="image" src="https://github.com/apache/airflow/assets/33751805/e3638098-2015-434b-b7a4-8ad61c0af005">